hack/build-rpm-release.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script generates release zips and RPMs into _output/releases.
+# tito and other build dependencies are required on the host. We will
+# be running `hack/build-cross.sh` under the covers, so we transitively
+# consume all of the relevant envars.
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+os::build::setup_env
+os::util::environment::setup_tmpdir_vars "build-rpm-release"
+
+if [[ "${OS_ONLY_BUILD_PLATFORMS:-}" == 'linux/amd64' ]]; then
```

Comment:
It's fine for now.  I don't think it makes a ton of sense in the future,  but can live with it.